docs/reference/schema-lang.md

```diff
@@ -254,6 +254,76 @@ It is **highly** recommended to put doc comments on all definitions, relations a
 
 <InlinePlayground reference="Vwput-WCrLaz"/>
 
+## Common Patterns
+
+### Group membership
+
+Apply specific users or members of a group to a permission on an object type.
+
+<details>
```

Comment:
Polled the room and it sounds like we don't want to hide this stuff inside `<details>`.